packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -46,6 +46,8 @@ export const messages = {
     'SFDX: Authorize an Org using Session ID',
   force_auth_access_token_login_bad_oauth_token_message:
     'The session ID that you are trying to use is not valid. Check if it has expired, or use a valid session ID.',
+  force_auth_web_login_device_code_parse_error:
+    'There was an unexpected error authorizing to you org in a container environment.',
```

Comment:
This is the only new change in this file.

Reply:
'to your org'

Reply:
good catch, thanks.